test/Handlebars.Net.Helpers.Tests/Helpers/DateTimeHelpersTests.cs

```diff
@@ -85,4 +85,240 @@ public void Format_With_ValueAsString(string format, string expected)
         // Assert
         result.Should().Be(expected);
     }
+
+    [Theory]

```

Comment:
Can you also add some real template tests?
See "DateTimeHelpersTemplateTests.cs"

Also for object?